docs: updates for azure resource-group property. (#593)
nitrictech/docs: src/pages/reference/providers/azure/configuration.mdx

# https://azure.microsoft.com/en-us/explore/global-infrastructure/products-by-region/?products=container-apps
region: my-azure-stack-region

# Deploy into an existing resource group
# Available since v1.11.0

> # ⚠️ Changes to this property will against a currently deployed stack is a destructive operation

Reviewer: ```suggestion
# ⚠️ Changing this property post-deployment is a destructive operation
```
